modules/oe_content_entity/modules/oe_content_entity_venue/src/Entity/Venue.php

```diff
@@ -69,4 +70,17 @@
  *  type = {"entity"}
  * )
  */
-class Venue extends CorporateEntityBase implements VenueInterface {}
+class Venue extends CorporateEntityBase implements VenueInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
+    $fields = parent::baseFieldDefinitions($entity_type);
+
+    $fields['name']->setRequired(FALSE);
```

Comment:
We can't set the entity label not required.

Reply:
Fixed : I let the field required but waiting [this issue ](https://www.drupal.org/project/inline_entity_form/issues/3099844) is fixed what is your suggestion for our usage ?
- Change Form display Widget ?
- Propose a patch for the issue ?

At this moment I updated the widget **Inline entity form - Simple** with **Inline entity form - Complex**